.github/workflows/tox.yml

```diff
@@ -0,0 +1,74 @@
+# yamllint disable rule:line-length
+name: tox
+on:  # yamllint disable-line rule:truthy
+  - pull_request
+jobs:
+  python:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pyver: ['2.7', '3.6', '3.7', '3.8']
+    steps:
+      - name: checkout PR
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.pyver }}
+      - name: Install platform dependencies
+        run: |
+          set -euxo pipefail
+          .travis/preinstall
```

Comment:
The preinstall script should probably be moved into .github/workflows

Reply:
1) The purpose of this PR is to move as quickly as possible from travis to github actions to cover the gap in CI coverage left when travis pulled the plug on open source projects.  So I don't want to make any other changes in this PR.
2) This will be provided by tox-lsr, which will hopefully be following once github actions are working